frontend/src/actions/resources.js

```diff
@@ -214,7 +214,7 @@ const mostRelevant = (options) => {
 };
 
 const countries = {
-  USA: "United States of America",
+  USA: "United States",
```

Comment:
The user can use both the widget or autofill for the country field. Therefore, you should check if the country field is "United States of America" or "United States".